src/Sparrow/LowMemory/MemoryInfoResult.cs

```diff
@@ -2,19 +2,6 @@
 {
     public struct MemoryInfoResult
     {
-        public class MemoryUsageIntervals
-        {
-            public Size LastOneMinute;
-            public Size LastFiveMinutes;
-            public Size SinceStartup;
-        }
-
-        public class MemoryUsageLowHigh
```

Comment:
I wonder if the removal of those can cause any binary compatibility issues. For the sake of safeness let's leave them here but mark as obsolete.

Reply:
Fixed